src/main/java/dan200/computercraft/shared/computer/apis/CommandAPI.java

```diff
@@ -98,6 +99,7 @@ public void shutdown()
             }
             catch( Throwable t )
             {
+                ComputerCraft.log.error( "Error running command.", t );
```

Comment:
A modification to the idea, each computer/peripheral has a max rate of logging per minute.